src/lib/types.ts

```diff
@@ -38,6 +38,12 @@ export type SupabaseClientOptions<SchemaName> = {
      * A storage provider. Used to store the logged-in session.
      */
     storage?: SupabaseAuthClientOptions['storage']
+    /**
+     * A storage provider to store the user profile separately from the session. Useful when you need to store the session information in cookies, without bloating the data with the redundant user object.
```

Comment:
```suggestion
     * A storage provider to store the user profile separately from the session.
     * Useful when you need to store the session information in cookies,
     * without bloating the data with the redundant user object.
```

just to make it look prettier